### PR TITLE
Add contribution guide

### PR DESCRIPTION
**Pull request by samgoldman**

### CODE REVIEW

**Review by Younes3807 — APPROVED** [review]
Good job!

File: CONTRIBUTIONS.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+Enter a descriptive commit message and stage the changes:
+
+![Committing in VSCode](images/vscode_committing.png)
+
+Press the `+` button for each file you would like to commit. Tip: you can (and should) click on each file in the commit menu to review the changes you're making before committing them.
+
+## 6. Git sync
+
+Once you've make the commit, you should sync it to GitHub. In the source control panel click 'Publish Branch' (if this is a new branch) or 'Sync Changes' (if you've already published the branch). You should do this frequently (ideally after every commit, even if you're not ready) so that others can view your changes and to back up your changes in case something happens to your copy of the repository.
```
Comment (Yathaarth3807) [inline CONTRIBUTIONS.md:57]: Typo on the third word, should be "made"

**Review by Yathaarth3807 — CHANGES_REQUESTED** [review]

File: CONTRIBUTIONS.md (hunk 1 above)
Comment (samgoldman) [reply CONTRIBUTIONS.md:57]: Oops, I'll fix that!

Comment (samgoldman) [reply CONTRIBUTIONS.md]: Fixed!

**Review by Younes3807 — APPROVED** [review]
Good Job! (again :>)

**Review by Yathaarth3807 — APPROVED** [review]
Looks good!

**Pull request merged**